src/linkify-html.js

```diff
@@ -109,7 +109,9 @@ function linkifyChars(str, opts) {
 			tagName,
 			className,
 			target,
-			attributes
+			attributes,
+			truncate,
+			truncateSize
```

Comment:
Hi @willamesoares, apologies for the delay on this. Rather than having the two options, can you keep it to a single `truncate: number` option? If you leave `truncate` unspecified or set to `0`, it'll leave the link unchanged.

Reply:
Hi, I'm okay with that. What would be the default value for `truncate`?

Reply:
Nvm, I guess `0` shall be :)